drivers/ethernet/Kconfig

```diff
@@ -55,5 +55,6 @@ source "drivers/ethernet/Kconfig.native_posix"
 source "drivers/ethernet/Kconfig.stellaris"
```

Comment:
split this patch in parts: 
- one that add the dsa core part (include/net/ and drivers/ethernet/dsa)
- one for net core specifics

Reply:
Could you be more specific here?
I do guess that you refer to: `eth: Add support for Distributed Switch Architecture [DSA] switches` patch.
How shall it be split?